data/nfv/vsperf_sle12.4.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+zypper -q -n in -y $(echo "
+# compiler, tools and dependencies
+make
+automake
+gcc
+gcc-c++
+glibc
+glibc-devel
+fuse
+fuse-devel
+glib2-devel
+zlib-devel
+kernel-default
+kernel-default-devel
+pkg-config
+findutils-locate
+curl
+automake
+autoconf
+vim
+wget
+git-core
+pciutils
+cifs-utils
+socat
+sysstat
+java-1_8_0-openjdk
+mlocate
+
+python-setuptools
+python3-setuptools
+python3-devel
+
+# libraries
+libnuma1
+libnuma-devel
+libpixman-1-0
+libpixman-1-0-devel
+libtool
+libpcap-devel
+libnet9
+libncurses6
+libcurl4
+libcurl-devel
+libxml2
+libfuse2
+libopenssl1_0_0
+libopenssl-devel
+libpython3_4m1_0
+libzmq3
+
+" | grep -v ^#)
+
+wget -q http://download.opensuse.org/repositories/openSUSE:/Backports:/SLE-12/standard/x86_64/sshpass-1.06-2.1.x86_64.rpm
+rpm -i sshpass-1.06-2.1.x86_64.rpm
```

Comment:
Why would you need ssh-pass? You could just exec_and_insert_password() from your test code that does what you want

Reply:
sshpass is a package requirement needed by VSPERF tool. If I don't instal it, the installation will complain

Reply:
The tool still needs shhpass for some internal mechanism, not sure where. I will try in the future if we can avoid it.